### PR TITLE
SSheetDialog删除@key，解决关闭后需点击两次才打开弹窗

### PR DESCRIPTION
**Pull request by zyz**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿@inherits PmCompontentBase
 
-<SSheetDialog @key="@($"app-{_appFormModel.Visible}")" ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Icon="mdi-circle" IconColor="success">
+<SSheetDialog ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Icon="mdi-circle" IconColor="success">
```
Comment (codding-y) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor:3]: Add 'EnableDomReload' prop

File: src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor [hunk 2]
```diff
@@ -94,7 +94,7 @@
 
 </div>
 
-<SSheetDialog @key="@($"env-{_envFormModel.Visible}")" ValueChanged="EnvModalValueChanged" Value="_envFormModel.Visible" Title="@(_envFormModel.HasValue ? $"{T("Edit Environment")}":$"{T("Add Environment")}")" Icon="mdi-circle" IconColor="success">
+<SSheetDialog ValueChanged="EnvModalValueChanged" Value="_envFormModel.Visible" Title="@(_envFormModel.HasValue ? $"{T("Edit Environment")}":$"{T("Add Environment")}")" Icon="mdi-circle" IconColor="success">
```
Comment (codding-y) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor:97]: Add 'EnableDomReload' prop

File: src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor [hunk 3]
```diff
@@ -158,7 +158,7 @@
         </MForm>
 </SSheetDialog>
 
-<SSheetDialog @key="@($"cluster-{_clusterFormModel.Visible}")" ValueChanged="ClusterModalValueChanged" Value="_clusterFormModel.Visible" Title="@(_clusterFormModel.HasValue ? $"{T("Edit Cluster")}":$"{T("Add Cluster")}")" Icon="mdi-circle" IconColor="success">
+<SSheetDialog ValueChanged="ClusterModalValueChanged" Value="_clusterFormModel.Visible" Title="@(_clusterFormModel.HasValue ? $"{T("Edit Cluster")}":$"{T("Add Cluster")}")" Icon="mdi-circle" IconColor="success">
```
Comment (codding-y) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor:161]: Add 'EnableDomReload' prop

File: src/Web/MASA.PM.Web.Admin/Pages/Home/ProjectModal.razor [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿@inherits PmCompontentBase
 
-<SSheetDialog @key="@($"project-{_projectFormModel.Visible}")" Value="_projectFormModel.Visible" ValueChanged="ProjectModalValueChanged" Title="@(_projectFormModel.HasValue ? $"{T("Edit Project")}":$"{T("Add Project")}")" Icon="mdi-circle" IconColor="success">
+<SSheetDialog Value="_projectFormModel.Visible" ValueChanged="ProjectModalValueChanged" Title="@(_projectFormModel.HasValue ? $"{T("Edit Project")}":$"{T("Add Project")}")" Icon="mdi-circle" IconColor="success">
```
Comment (codding-y) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/ProjectModal.razor:3]: Add 'EnableDomReload' prop

**Review by codding-y — CHANGES_REQUESTED** [review]

File: src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor (hunk 1 above)
Comment (zyz) [reply src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor]: The latest main branch code has been merged，now use SSimpleModal, without 'EnableDomReload'

**Comment by zyz** [comment]
The latest main branch code has been merged, now use 'SSimpleModal',  without 'EnableDomReload'

**Review by codding-y — APPROVED** [review]

**Pull request merged**